training/model_management/components/model_sanity_validation/spec.yaml

```
@@ -0,0 +1,49 @@
+$schema: https://azuremlschemas.azureedge.net/latest/commandComponent.schema.json
+
+name: mlflow_model_local_validator
+version: 0.0.1
+type: command
+
+is_deterministic: True
+
+display_name: MLFlow model local validator
+description: Validates if a MLFLow model is locally loadable and inferencable
+
+environment: azureml:model-management:1
+
+code: ../../src/
+command: |
+  set -ex
+  export ENV_PREFIX="/opt/conda/envs/inferencing"
+  echo $ENV_PREFIX
+  ls -ltr ${{inputs.model_path}}
+  cp ${{inputs.model_path}}/conda.yaml ./
+  conda env create -p $ENV_PREFIX -f conda.yaml -q
+  echo "conda env successfully created at $ENV_PREFIX"
+  conda run -p $ENV_PREFIX python run_model_sanity_validations.py --model-path ${{inputs.model_path}} --test-data-path ${{inputs.test_data_path}} $[[--column-rename-map "${{inputs.column_rename_map}}"]] --output-model-path ${{outputs.mlflow_output_path}}
+
+inputs:
+  model_path:
+    type: mlflow_model
+    mode: ro_mount
+    description: Input mlflow model for sanity validations
+
+  test_data_path:
+    type: uri_file
+    description: Test dataset for model inference validation
```

Could be simplified. Also, should the customer get any additional info about the dataset or its format?
```suggestion
    description: Test dataset
```